sdk/core/azure-core/src/main/java/com/azure/android/core/http/interceptor/LogUtils.java

```diff
@@ -40,33 +30,45 @@ static long getContentLength(Headers headers) {
      * @return "Log body" if the body should be logged in its entirety, otherwise a message indicating why the body
      * was not logged is returned.
      */
-    static String evaluateBody(Headers headers) {
+    @SuppressWarnings("ConstantConditions")
```

Comment:
What was causing this warning? It doesn't seem like it should be triggering.

Reply:
Going to check out this PR and see what the heck is going on there because I can't see any reason why that would generate this warning.